gratipay/models/team.py

```diff
@@ -67,6 +67,26 @@ def insert(cls, owner, **fields):
         """, fields)
 
 
+    def update(self, **kw):
+      updateable = frozenset(['name', 'product_or_service', 'homepage',
+                              'onboarding_url', 'todo_url'])
+
+      cols, vals = zip(*kw.items())
+      assert set(cols).issubset(updateable)
```

Comment:
It won't receive extra columns, at least not from the web endpoint. https://github.com/gratipay/gratipay.com/pull/3923/files#diff-8cd27f320d80ef3f1fed6dc52f5feafdR40 makes sure of that. This check is just in case this is called from somewhere else in future in which case they should catch the exception.